AzureMonitor: User authentication support (#81918)
grafana/grafana: docs/sources/setup-grafana/configure-grafana/_index.md

Disabled by default, needs to be explicitly enabled.

### user_identity_fallback_credentials_enabled

> Specifies whether user identity authentication fallback credentials should be enabled in data sources. Enabling this allows data source creators to provide fallback credentials for backend initiated requests e.g. alerting, recorded queries etc.

Reviewer: ```suggestion
Specifies whether user identity authentication fallback credentials should be enabled in data sources. Enabling this allows data source creators to provide fallback credentials for backend-initiated requests, such as alerting, recorded queries, and so on.
```

> Enabled by default, needs to be explicitly disabled. Will not have any effect if user identity is disabled above.

Reviewer: ```suggestion
It is enabled by default and needs to be explicitly disabled. It will not have any effect if the user identity is disabled.
```
